tgui/packages/tgui/interfaces/OreRedemptionMachine.js

```diff
@@ -1,107 +1,209 @@
+                    <Icon
+                      name="id-card"
+                      size={3}
+                      mr={1}
+                      color={user ? 'green' : 'red'}
+                    />
+                  </Stack.Item>
+                  <Stack.Item>
+                    {(!user && 'No user Detected') || (
+                      <LabeledList>
+                        <LabeledList.Item label="Name">
+                          {user.name || 'No name detected'}
+                        </LabeledList.Item>
+                        <LabeledList.Item label="Balance">
+                          {user.cash + ' cr' || 'No balance detected'}
+                        </LabeledList.Item>
+                      </LabeledList>
+                    )}
+                  </Stack.Item>
+                </Stack>
+              </Section>
+            </Stack.Item>
+          </Section>
+          <Section>
+            <Stack.Item>
+              <Box>
+                <Icon name="coins" color="gold" />
+                <Box inline color="label" ml={1}>
+                  Unclaimed points:
+                </Box>
+                {unclaimedPoints}
+                <Button
+                  ml={2}
+                  content="Claim"
+                  disabled={unclaimedPoints === 0}
+                  onClick={() => act('Claim')}
+                />
+              </Box>
+            </Stack.Item>
+          </Section>
+          <Section>
+            <Stack.Item>
+              <BlockQuote>
+                This machine only accepts ore. Gibtonite and Slag are not
+                accepted.
+              </BlockQuote>
+            </Stack.Item>
+          </Section>
+          <Tabs>
+            <Tabs.Tab
+              icon="list"
+              lineHeight="23px"
+              selected={tab === 'material'}
+              onClick={() => {
+                setTab('material');
+
+                if (searchItem.length > 0) {
+                  setSearchItem('');
+                }
+              }}>
+              Materials
+            </Tabs.Tab>
+            <Tabs.Tab
+              icon="list"
+              lineHeight="23px"
+              selected={tab === 'alloy'}
+              onClick={() => {
+                setTab('alloy');
+
+                if (searchItem.length > 0) {
+                  setSearchItem('');
                 }
-              />
-            ))}
-          </Table>
-        </Section>
-        <Section title="Alloys">
-          <Table>
-            {alloys.map((material) => (
-              <MaterialRow
-                key={material.id}
-                material={material}
-                onRelease={(amount) =>
-                  act('Smelt', {
-                    id: material.id,
-                    sheets: amount,
-                  })
+              }}>
+              Alloys
+            </Tabs.Tab>
+            <Input
+              autofocus
+              ml={22}
+              mb={0.8}
```

Comment:
Instead of this, you should add position="relative" to Tabs and use `top` and `right` props here to anchor the input; just no margins.